importer/Cargo.toml

```diff
@@ -17,7 +17,7 @@ collisions = { path = "../collisions" }
 convert_osm = { path = "../convert_osm" }
 csv = "1.1.4"
 geo = "0.17.0"
-geojson = "0.22"
+geojson = { version = "0.22.0", features = ["geo-types"] }
```

Comment:
This feature is being used in this crate, but previously it was being silently added from `popdat`

Reply:
https://blog.rust-lang.org/2021/05/11/edition-2021.html#default-cargo-feature-resolver
I wasn't, but awesome! Maybe we ought to opt into some of this sooner.